app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -4,38 +4,35 @@ class GithubTag
   class GithubReadmeTag
     PARTIAL = "liquids/github_readme".freeze
 
+    attr_reader :client, :content, :options, :readme_html
+
     def initialize(link)
-      @link = parse_link(link)
+      parsed_link = parse_link(link)
       @options = parse_options(link)
-      @content = get_content(@link)
+      @client = Octokit::Client.new(access_token: token)
+      @content = client.repository(parsed_link)
+      @readme_html = fetch_readme(parsed_link)
```

Comment:
I think `readme_html` is a much better name for this, the fact that it was updated (the old name was `updated_html` is really more of an implementation detail.